### PR TITLE
`openpyxl`: type `tagname`, `namespace` and `__doc__` ClassVar and params

### PR DESCRIPTION
**Pull request by Avasam**
`tagname` is always set as a `ClassVar`. The only moment it's not is when it's not been overridden, in which case it's a property that throws `NotImplementedError` but shouldn't make the class abstract.

`namespace` is set as a `ClassVar` (either as some literal string or None), except for `Nested` descriptors, where it it an optional param in `__init__`.

### CODE REVIEW

File: stubs/openpyxl/@tests/test_cases/check_nested_descriptors.py
Comment (srittau) [inline stubs/openpyxl/@tests/test_cases/check_nested_descriptors.py:35]: This can't really work as a type annotation, can it? (Same for the existing attributes below.)

Comment (Avasam) [reply stubs/openpyxl/@tests/test_cases/check_nested_descriptors.py:35]: It"s not a type annotation, it's an actual assignement.

**Review by srittau — APPROVED** [review]
Good to go, once the merge conflicts are resolved.

**Pull request merged**
